### DISCUSSION TITLE
Using VideoTexture for MpPoseTracking

### DISCUSSION THREAD

**Discussion by fariskas** [cables-gl/cables_docs#539] [Answers & Questions]
Hello.

I'm trying to use a video texture for the Mediapipe Pose Tracking op, but it seems to only take a Webcam html element.

<img width="571" alt="Screen Shot 2023-06-15 at 6 41 33 PM" src="https://github.com/cables-gl/cables_docs/assets/24261675/d9002754-e3db-4962-8fd3-765c2bc42387">

Alternatively i tried to make a video element instead but the problem with this is that this does not provide a texture to use on other elements.

Is there a way to directly pass a videoTexture to the mpPoseTracking op instead?

**Comment by 60-hz** [comment]
Same issue here. A working workaround is to use OBS, create a video source, and then make a virtual webcam.

**Comment by pandrr** [comment]
<img width="627" alt="Screenshot 2024-04-21 at 10 13 42" src="https://github.com/cables-gl/cables_docs/assets/98792/c2feb87c-7e04-47bb-be82-919bc81ac456">

on dev.cables.gl you could use those new ops to convert the videoelement to a texture


thinking about it, in the future, the canvasToTexture should be a "imageElement" to texture and do this directly....

**Comment by pandrr** [comment]
<img width="530" alt="Screenshot 2024-04-22 at 14 08 33" src="https://github.com/cables-gl/cables_docs/assets/98792/ee648619-ee20-4b27-9902-a0c0ec19196c">

this is now possible on dev.cables.gl
